client/src/app/primitives/modal/Modal.js

```diff
@@ -53,13 +53,14 @@ export default class Modal extends PureComponent {
 
     const {
       className,
-      children
+      children,
+      onClose
     } = this.props;
 
     return ReactDOM.createPortal(
       <div className={ css.ModalOverlay } onClick={ this.handleBackgroundClick }>
         <div className={ classNames(css.ModalContainer, className) } ref={ this.modalRef }>
-          <Close onClick={ this.close } />
+          { onClose && (<Close onClick={ this.close } />) }
```

Comment:
This does break background clicks now, right? `this.close` does always assume the property is there.

Reply:
Ah yep, it did for the PrivacyPreferencesSettings, fixed it now.